.github/workflows/build-release.yml

```diff
@@ -62,12 +59,17 @@ jobs:
         with:
           toolchain: 1.37.0
           override: true
+      
+      - name: Test version
+        run: echo ${{steps.get_version.outputs.VERSION}}
+        shell: bash
+
```

Comment:
Is this left over from debugging? AFAICT it shouldn't output anything because the `get_version` step hasn't run yet.